src/routes/game/components/dialogs/GameDialogs.vue

```diff
@@ -46,14 +46,14 @@
 import { mapStores } from 'pinia';
 import { useGameStore } from '@/stores/game';
 import { useAuthStore } from '@/stores/auth';
-import CannotCounterDialog from '@/routes/game/components/CannotCounterDialog.vue';
-import CounterDialog from '@/routes/game/components/CounterDialog.vue';
-import FourDialog from '@/routes/game/components/FourDialog.vue';
-import GameOverDialog from '@/routes/game/components/GameOverDialog.vue';
-import ReauthenticateDialog from '@/routes/game/components/ReauthenticateDialog.vue';
-import SevenDoubleJacksDialog from '@/routes/game/components/SevenDoubleJacksDialog.vue';
-import ThreeDialog from '@/routes/game/components/ThreeDialog.vue';
-import OpponentRequestedStalemateDialog from '@/routes/game/components/OpponentRequestedStalemateDialog.vue';
+import CannotCounterDialog from './components/CannotCounterDialog.vue';
```

Comment:
We really should be using the `@` syntax as much as we can. If you need them shorter you could make an `gameComponents` alias or something.

Reply:
So counterpoint: I've reorganized the components directory to better represent parent/child relationships. I see this as an extension of how we already colocated page files with their constituent components. This basically means that for any given component, its children will always be in `./components`. 

I think that actually makes it more intuitive to find what you're looking for, because everything is organized from the page level down and any given thing should always be able to find it's direct children in `./components`. I don't have strong feelings about it, but I do like the look of everything getting to use ./components as its succinct, logical, and gels with the possibility of having ./composables, etc

Reply:
I’m certainly not going to push the point, I just never really like mixing those styles but I agree that it’s pretty nice for parent/sibling refs.

Reply:
good enough for now